### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by Hywan**
Sequel of https://github.com/mozilla/vaani/pull/68#issuecomment-150898640.

### CODE REVIEW

File: src/localization/fr-FR.l20n [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  <p>(bientôt disponible)</p>
+">
+
+
+<standingBy__howMayIHelpYou "Comment puis-je aider ?">
+<standingBy__iCantFindThatApp "Je ne trouve pas {{ $app }}."
+  spoken: "Je ne trouve pas cette application."
+>
+<standingBy__openCommand "ouvre">
+<standingBy__openCommandCue "démarre"> /* enum: start | end */
```
Comment (jedireza) [inline src/localization/fr-FR.l20n]: This should be either the literal strings `start` or `end`. We use this to determine if we should detect this command in the beginning or end of a string. See: https://github.com/mozilla/vaani/blob/d822f46115c7f43b6d142d7b3ad8df6f7e0e0ed3/src/actions/standing-by.js#L135

File: src/localization/fr-FR.l20n [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+<standingBy__callCommand "appelle">
+<standingBy__callCommandCue "démarre"> /* enum: start | end */
```
Comment (jedireza) [inline src/localization/fr-FR.l20n]: This should be either the literal strings `start` or `end`.

File: src/localization/fr-FR.l20n [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+<standingBy__dialCommand "compose">
+<standingBy__dialCommandCue "démarre"> /* enum: start | end */
```
Comment (jedireza) [inline src/localization/fr-FR.l20n]: This should be either the literal strings `start` or `end`.

File: src/localization/fr-FR.l20n [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+<standingBy__specialAppPhone "téléphone">
+<standingBy__specialAppContacts "contacts">
```
Comment (jedireza) [inline src/localization/fr-FR.l20n:58]: We should make sure that the apps for contacts and phone match these values exactly when French is activated. If not, they won't be found and those apps won't open.

**Comment by jedireza** [comment]
This is great. Thanks so much for creating a PR. I left a couple comments in-line.

Also, I noticed you renamed `src/localization/fr.l20n` => `src/localization/fr-FR.l20n`, which is fine. To finish this, we'll need to update the localizer's `supportedLocales` array too. See https://github.com/mozilla/vaani/blob/d822f46115c7f43b6d142d7b3ad8df6f7e0e0ed3/src/lib/localizer.js#L17

File: src/localization/fr-FR.l20n (hunk 4 above)
Comment (Hywan) [inline src/localization/fr-FR.l20n:58]: Where can we found this information?

**Comment by Hywan** [comment]
@jedireza Done. Just one question left (hidden by Github): https://github.com/mozilla/vaani/pull/69#discussion_r42976364.

File: src/localization/fr-FR.l20n (hunk 4 above)
Comment (jedireza) [inline src/localization/fr-FR.l20n:58]: My mistake, we already handle this. :grin: 

https://github.com/mozilla/vaani/blob/d822f46115c7f43b6d142d7b3ad8df6f7e0e0ed3/src/actions/standing-by.js#L208-L217

**Comment by jedireza** [comment]
@Hywan my mistake, see reply https://github.com/mozilla/vaani/pull/69#discussion_r43081298

When checking this out I found another bug and created PR #70, so it's good we double checked.

**Comment by jedireza** [comment]
@andrenatal would you like to give this grammar a test drive before we merge?

**Comment by jedireza** [comment]
Also @Hywan let's squash the commits into one. If you need help with squashing commits see:
- https://github.com/ginatrapani/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit
- http://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#Squashing-Commits

**Comment by andrenatal** [comment]
@jedireza Yes, let me run it with French models and see if compiles well.

**Comment by Hywan** [comment]
@jedireza I know how to squash, thanks! Do you have a specific guide line for the commit message?
I wait on @andrenatal's feedback before squashing everything.

**Comment by jedireza** [comment]
Perfect. The first commit message sounds good to me `Add French translations.`.

**Comment by jedireza** [comment]
Wow this is getting old. We should try to wrap this up.

@andrenatal What's the latest based on your last message?
@Hywan Squashing the commits isn't a bad move, nothing should stop us from merging. Your PR puts French is much better shape than we have now.

Let's :shipit: y'all.

**Comment by Hywan** [comment]
Still relevant?

**Comment by kdavis-mozilla** [comment]
No. There is no active development on project Vaani and also on FirefoxOS.

**Pull request closed without merging**